rbac/default-role-manager/role_manager_test.go

```diff
@@ -54,12 +54,12 @@ func testPrintRoles(t *testing.T, rm rbac.RoleManager, name string, res []string
 
 func TestRole(t *testing.T) {
 	rm := NewRoleManager(3)
-	rm.AddLink("u1", "g1")
-	rm.AddLink("u2", "g1")
-	rm.AddLink("u3", "g2")
-	rm.AddLink("u4", "g2")
-	rm.AddLink("u4", "g3")
-	rm.AddLink("g1", "g3")
+	_ = rm.AddLink("u1", "g1")
```

Comment:
`AddLink` returns an error. This explicitly indicates the error is being ignored.
Otherwise `errcheck` via [gometalinter](https://github.com/alecthomas/gometalinter) produces the following:
```
rbac/default-role-manager/role_manager_test.go:57:12:warning: error return value not checked (rm.AddLink("u1", "g1")) (errcheck)
```

Reply:
Given it is not a Golang official advice, I tend not to follow it. Because it makes a lot of code look tedious and doesn't help readers to understand the code better. Can you rework the commit by removing such changes? Thanks.

Reply:
The Golang official advice would be to actually check each of the errors:
https://github.com/golang/go/wiki/CodeReviewComments#handle-errors
https://github.com/golang/lint

To keep backwards compatibility I took the path of explicitly ignoring the errors.

Reply:
Handling each error is not practical. My thought is to keep the code as simple as possible, but leave users the option to make it robust (and complicated at most times). 

>To keep backwards compatibility I took the path of explicitly ignoring the errors.

Could the commit be improved?